lib/Model/DataObject/ClassDefinition/ClassLayoutDefinitionManager.php

```diff
@@ -26,12 +26,20 @@ class ClassLayoutDefinitionManager
     public const DELETED = 'deleted';
 
     /**
+     * @deprecated
+     *
      * Delete all custom layouts from db
      */
     public function cleanUpDeletedLayoutDefinitions(): array
     {
         $db = \Pimcore\Db::get();
-        $layouts = $db->fetchAllAssociative('SELECT * FROM custom_layouts');
+
+        try {
+            $layouts = $db->fetchAll('SELECT * FROM custom_layouts');
```

Comment:
```suggestion
            $layouts = $db->fetchAllAssociative('SELECT * FROM custom_layouts');
```
The `fetchAll` is deprecated and were replaced in https://github.com/pimcore/pimcore/issues/12363

Reply:
good catch! 😉